docs/eks-installation.md

```diff
@@ -0,0 +1,34 @@
+# Deploying Antrea in AWS EKS
+
+This document describes steps to deploy Antrea in NetworkPolicy only mode to an AWS EKS cluster.
+
+Assuming you already have an EKS cluster, and have KUBECONFIG environment variable pointing to<br>
+the kubeconfig file of that cluster.
+
+Based on EKS worker Node MTU size and service CIDR, adjust defaultMTU and serviceCIDR values of<br>
```

Comment:
Better to make it more clear, like "Kubernetes Service Cluster IP range"

Reply:
Highlight `defaultMTU` and `serviceCIDR`.

Reply:
Should we mention MTU of EKS VMs is typically 9001? Maybe better to add a yaml path to change the default MTU to 9001 too.

Reply:
+1 if we know the typical MTU for EKS nodes, then let's use that (minus encap overhead) as the defaultMTU parameter in antrea-eks.yml.

Once this PR is merged, we can also look into automatic MTU detection

Reply:
Leave MTU size untouched because aws ec2 mtu is configurable. 

Rest is done.